book/survival.qmd

```diff
@@ -212,79 +244,85 @@ This data would be collected on a yearly scale and over three years may look som
+To create a matched cohort, two other infants, whose mothers were alive, were matched into the study based on their age and other relevant features.
+Thus, groups of three infants within the study had identical features except for the status of the mother (alive or dead).
+Because of the study design, infants who died before their mothers could never enter into the study. A mother's death is thus refered to as left-truncation event and the infant's age at time of inclusion into the study is refered to as left-truncation time.
 
-In a study  from the 18th century @brostrom.influence.1987, when childhood and maternal mortality were relatively high, data was collected on infant survival during the first year after birth based on whether the mother was alive.
-Data was collected by adding an infant to the study when their mother died, then two other infants, whose mothers were alive, were matched into the study based on their age and features.
-The age of the infant at entrance into the study therefore signifies a *left-truncation event*, as infants who die before their mothers never enter the data and the *left-truncation time* is given by the infants' age at which the mother died.
+More formally, let $t^L_i$ the subject-specific left-truncation time.
+Then we only observe subjects with $y_i > t^L_i$ and subjects with $y_i < t^L_i$ never enter the data.
 
-More formally, let $T^l_i$ the subject-specific left truncation time.
-Then we only observe subjects with $Y_i > T^l_i$ or $C_i > T^l_i$.
-Subjects with $Y_i < T^l_i$ never enter the data.
+This is illustrated in @fig-left-truncation.
+Continuing with the example above, say Infant 1 dies at $t_1$, while the mother dies at some later timepoint $t_1^L$, thefore Infant 1 never enters the study. 
+The mother of Infant 2 dies at $t_2^L$, at which point the infant is included in the study and experiences an event at $t_2$.
+Finally, say Infant 3 enters the study at $t_3^L$ and is censored at 365 days when the study ends.
 
-Notably, left-truncation also plays an important role when modeling recurrent events or multi-state data, as the data generating process induces left-truncation (@sec-eha).
+![Illustration of left-truncation data. Subjects 2 and 3 enter the data set as the study entry condition (mother's death) occurs before the event of interest occurs after the left-truncation time (left-truncation time shorter than event time). Subject 1 on the other hand never enters the data as the event occurs before the study entry condition (left-truncation time longer than event time).](Figures/survival/left-truncation.png){#fig-left-truncation fig-alt="Schematic illustration of left-truncation."}
 
+Left-truncation plays an important role when modeling recurrent events or multi-state data, as the data generating process induces left-truncation (@sec-eha).
 
-#### Right-truncation {.unnumbered .unlisted}
+
+### Right-truncation {.unnumbered .unlisted}
```

Comment:
I think the example below needs better explanation of what $t_i^r$ is, I'm a bit confused. Is it the difference between query time (say tau) and the disease onset time? If so maths would be useful. Again below needs to be clearer how to handle this case and also the bias (although there is more detail here)